src/@lekoarts/gatsby-theme-cara/components/SpeakerCard.tsx

```diff
@@ -47,6 +49,24 @@ const SpeakerCard = ({ name, children, bg, imageSrc }: SpeakerCardProps) => (
     >
       {name}
     </div>
+    <div 
+      sx={{
+        letterSpacing: `wide`,
+        fontSize: [1, 2],
+        lineHeight: 1,
+      }}>
```

Comment:
Keep  `>` in the next line
```suggestion
      }}
    >
```